### ISSUE TITLE
[FeatureRequest] Azure Monitor per subscription quotas and usage

### ISSUE THREAD

**Issue by koceg** [grafana/grafana#30032]
**What would you like to be added**: Monitor quota consumption per subscription 

**Why is this needed**:  Alert on quota limit reach 

Hello, I've notices that currently Azure `SupportedNamespaces` is lacking some endpoints that provide per subscription overview.
what I'm after is what is explained here https://github.com/grafana/azure-monitor-datasource/issues/31#issuecomment-360672114_
At that point in time maybe the api was not available but now I see that this should be possible.

CLI Example:
```
az vm list-usage -l westeurope  # would list vCPU consumed per region/subscription
az storage account show-usage -l westeurope  # same as above but for storage
```
and these are the endpoints that get called to pull the needed data
API Example:
```
GET https://management.azure.com/subscriptions/{subscriptionId}/providers/Microsoft.Storage/locations/{location}/usages?api-version=2019-06-01
GET https://management.azure.com/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/usages?api-version=2020-06-01
GET https://management.azure.com/subscriptions/{subscriptionId}/providers/Microsoft.Network/locations/{location}/usages?api-version=2020-07-01 # emphasis on the pattern
```

Is this something that could be easily implemented?
Is this already implemented, but I'm using the wrong approach to get the needed data?

**Comment by papagian** [comment]
Which grafana version do you use?

**Comment by grafanabot** [comment]
Thanks for creating this issue! We think it's missing some basic information. 

Follow the issue template and add additional information that will help us replicate the problem. 
For data visualization issues: 
- Query results from the inspect drawer (data tab & query inspector)
- Panel settings can be extracted in the panel inspect drawer JSON tab

For dashboard related issues: 
- Dashboard JSON can be found in the dashboard settings JSON model view

For authentication and alerting issues, Grafana server logs are useful. 

Happy graphing!

**Comment by koceg** [comment]
> Which grafana version do you use?

version 7.3.3

**Comment by zuchka** [comment]
@koceg are you still encountering this issue?

**Comment by koceg** [comment]
Hello @zuchka, after creating this issue I implemented this in parallel [Prometheus azure exporter](https://github.com/koceg/azure.git) and forgot about it. Can be closed if low priority or already addressed is some other pull request.
